### PR TITLE
Allow user to create model using shortcuts

### PR DESCRIPTION
**Pull request by krishnahimself**
Hi,
Currently, there is an error while creating a model using shortcuts so this PR will fix that issue.
Error Message: The "-m" option does not exist.
Thanks

### CODE REVIEW

**Review by alexgaal — APPROVED** [review]

File: tests/Commands/MakeModelCommandTest.php [hunk 1]
```diff
@@ -31,3 +31,23 @@
         ->toBeFile()
         ->toPlaceholdersBeReplaced();
 });
+
+
+test('can make model, factory and migration are created using shortcuts', function () {
+    $this->artisan('beyond:make:model User/User -mf --force');
```
Comment (regnerisch) [inline tests/Commands/MakeModelCommandTest.php:37]: Is there a reason you are using the `force` flag?

Comment (krishnahimself) [reply tests/Commands/MakeModelCommandTest.php:37]: No. It should work without a force flag. Actually, I just followed the existing test by adding a shortcut flag.

Comment (regnerisch) [reply tests/Commands/MakeModelCommandTest.php:37]: Ah okay, die not see that :D

**Comment by regnerisch** [comment]
Hey, thank you for your contribution! :)

**Comment by krishnahimself** [comment]
You are welcome.

**Pull request merged**